lib/workers/repository/updates/flatten.ts

```diff
@@ -42,6 +42,14 @@ export function applyUpdateConfig(input: BranchUpgradeConfig): any {
         .replace(regEx(/^\//), '') // remove leading slash
         .replace(regEx(/\//g), '-') // change slashes to hyphens
         .replace(regEx(/-+/g), '-'); // remove multiple hyphens
+      updateConfig.sourceRepoOrg = parsedSourceUrl.pathname
+        .replace(regEx(/^\//), '') // remove leading slash
+        .replace(regEx(/\/.*/g), '') // remove everything after first slash
+        .replace(regEx(/-+/g), '-'); // remove multiple hyphens
+      updateConfig.sourceRepoName = parsedSourceUrl.pathname
+        .replace(regEx(/^\//), '') // remove leading slash
+        .replace(regEx(/^(.*?)\//g), '') // remove everything up to first slash
+        .replace(regEx(/-+/g), '-'); // remove multiple hyphens
```

Comment:
```suggestion
        .replace(regEx(/^(.*?)\//g), ''); // remove everything up to first slash
```

same here.

I think you need to split at last slash, as on gitlab we cvan have subgroups.

`group/sub/subsub/repo` => `sourceRepoOrg=group/sub/subsub` and `sourceRepoName=repo`